initdb/01-databases.sql

```diff
@@ -0,0 +1,6 @@
+# create databases
```

Comment:
instead of putting this into the new `initdb/` directory, how about just sticking this into the existing `fineract-db/` ?

Reply:
I moved this file to fineract-db/docker.